en/02_Developer_Guides/02_Controllers/02_Routing.md

```diff
@@ -303,6 +303,53 @@ class BreadAPIController extends Controller
+- There is a class `RequestHandlerTwo` with `$allowed_actions = ['hello']`, and a `hello()` method returning `HTTPResponse::create()->setBody('hello');`
+- Navigating to the URL `/one/two/hello` will return a response with a body of "hello"
+
+### Form, HasRequestHandler, FormRequestHandler, and FormField
+
+[`Form`](api:SilverStripe\Forms\Form) does not extend [`RequestHandler`](api:SilverStripe\Control\RequestHandler), instead it implements the [`HasRequestHandler`](api:SilverStripe\Control\HasRequestHandler) interface that requires a method `getRequestHandler()`. [`Form::getRequestHandler()`](api:SilverStripe\Forms\Form::getRequestHandler()) returns a [`FormRequestHandler`](api:SilverStripe\Forms\FormRequestHandler) which is a subclass of [`RequestHandler`](api:SilverStripe\Control\RequestHandler).
+
+Subclasses of [`RequestHandler`](api:SilverStripe\Control\RequestHandler) and implementors of [`HasRequestHandler`](api:SilverStripe\Control\HasRequestHandler) are treated the same because they will both end up calling `handleRequest()` on the subclass of the appropriate [`RequestHandler`](api:SilverStripe\Control\RequestHandler) subclass.
+
+[`FormRequestHandler`](api:SilverStripe\Forms\FormRequestHandler) includes a `$url_handler` entry `'field/$FieldName!' => 'handleField'` which allows it to handle requests to form fields on the form. [`FormRequestHandler::handleField()`](api:SilverStripe\Forms\FormRequestHandler::handleField()) will find the form field matching `$FieldName` and return it. Control is then passed to the returned form field.
+
+[`FormField`](api:SilverStripe\Forms\FormField) extends [`RequestHandler`](api:SilverStripe\Control\RequestHandler), which means that form fields are able to handle HTTP requests and have the `$allowed_actions` config. This allows form fields have define their own AJAX endpoints.
+
+### Example of an AJAX `FormField` that uses nested `RequestHandler`'s
+
+ The "Viewer groups" dropdown AJAX request when viewing file details in asset admin has an endpoint of `/admin/assets/fileEditForm/{FileID}/field/ViewerGroups/tree?format=json`
+
+- `assets` matches the `$url_segment = 'assets` of `AssetAdmin extends LeftAndMain`
+- `fileEditForm/{FileID}` matches the `'fileEditForm/$ID' => 'fileEditForm'` rule from `AssetAdmin` `$url_handlers`
+- `AssetAdmin::fileEditForm()` will return a `Form` scaffolded for the `File` matching the `FileID`
+- `Form::getRequestHandler()` will be called on the `Form` which returns a `LeftAndMainFormRequestHandler extends FormRequestHandler`
+- `field/ViewerGroups` will matches the `'field/$FieldName!' => 'handleField'` rule from `FormRequestHandler` `$url_handlers`
+- `FormRequestHandler::handleField()` will find the `ViewerGroups` field from the `Form`, the field is a `TreeMultiselectField extends TreeDropdownField`
+- `tree` will match the `$allowed_action` rule `tree` on `TreeDropdownField`
+- `TreeDropdownField::tree()` will return an `HTTPResponse` with its body containing JSON
```

Comment:
If this is intended to be the order of operations, it should be an ordered list, with a sentence before it that explains what the list is. Something like "That URL ends up passing the request through a series of nested request handlers. This is the path it takes:"